astropy_timeseries/tests/test_sampled.py

```diff
@@ -144,26 +149,36 @@ def test_fold():
     assert_allclose(tsf.time.sec, [-1.5, -0.5, 0.5, 1.5, -1.5, 1.5], rtol=1e-6)
 
 
-@pytest.mark.skipif('not HAS_PANDAS')
 def test_pandas():
+    pandas = pytest.importorskip("pandas")
 
     df1 = pandas.DataFrame()
     df1['a'] = [1, 2, 3]
     df1.set_index(pandas.DatetimeIndex(INPUT_TIME.datetime64), inplace=True)
 
     ts = TimeSeries.from_pandas(df1)
     assert_equal(ts.time.isot, INPUT_TIME.isot)
+    assert ts.colnames == ['time', 'a']
+
+    ts1 = TimeSeries.from_pandas(df1, index='a')
+    assert_equal(ts1.time.isot, INPUT_TIME.isot)
+    assert ts1.colnames == ['time', 'index', 'a']
```

Comment:
So in this case, does it mean that ``a`` and ``time`` are both index columns? Maybe you can test ``ts1.indices`` to check that the expected columns are indices?

Reply:
There was a mistake here, it should be a boolean here. 
I got confused when writing this test with to_pandas.

It just adds a copy of the time axis as a titled index column. 

Before:
```
ts
<TimeSeries length=3>
             time               a  
            object            int64
----------------------------- -----
2016-03-22T12:30:31.000000000     1
2015-01-21T12:30:32.000000000     2
2016-03-22T12:30:40.000000000     3
```

After

```
ts1
<TimeSeries length=3>
             time                      index            a  
            object                     object         int64
----------------------------- ----------------------- -----
2016-03-22T12:30:31.000000000 2016-03-22T12:30:31.000     1
2015-01-21T12:30:32.000000000 2015-01-21T12:30:32.000     2
2016-03-22T12:30:40.000000000 2016-03-22T12:30:40.000     3
```

If it is useful, I am not sure.